docs/guide/piece-pages.md

```diff
@@ -1,257 +1,162 @@
-module.exports = {
-  // ...
-  fields: {
-    add: {
-      // ... add this as one more field
-      color: {
-        type: 'select',
-        label: 'Color',
-        choices: [
-          {
-            value: 'red',
-            label: 'Red'
-          },
-          {
-            value: 'green',
-            label: 'Green'
-          },
-          {
-            value: 'blue',
-            label: 'Blue'
-          }
-        ]
-      }
-    }
-  }
-};
-```
+The pager macro is a special template using the [Nunjucks macro](https://mozilla.github.io/nunjucks/templating.html#macro) feature. This particular macro accepts two arguments:
 
-```js
-// modules/product-page/index.js
-module.exports = {
-  extend: '@apostrophecms/piece-page-type',
-  options: {
-    label: 'Product Index Page',
-    piecesFilters: [
-      {
-        name: 'color',
-        label: 'Color'
-      }
-    ]
-  }
-};
-```
+- an object with the `currentPage` and `totalPages` values, described above, as well as an optional CSS class for the pager wrapper
+- the page URL, using `data.url`
+
+Other than the class, the pager macro code above will rarely change.
+
+**You can change the number of pieces in each page of results** by setting [the `perPage` option](/reference/module-api/module-options.md#perpage-2) on the module. The data passed to templates will all update, so you don't need to make any other adjustments.
+
+## The show page template
+
+As a reminder, show pages are the web pages for individual pieces, rendered from `show.html` templates. This template uses a very standard page template structure. Assuming our `article` piece type example has a single `main` area, it could look like this:
 
 ```django
-{# modules/product-page/views/index.html #}
-
-{# ... add this before the list of products #}
-<nav>
-  {% for choice in data.piecesFilters.color %}
-    {% if data.query.color == choice.value %}
-      {# Click to remove the filter #}
-      {{ choice.label }}
-      <a
-        href="{{ data.url | build({ color: null }) }}"
-      >
-        ⓧ
-      </a>
-    {% else %}
-      {# Click to select the filter #}
-      <a
-        href="{{ data.url | build({ color: choice.value }) }}"
-      >
-        {{ choice.label }}
-      </a>
-    {% endif %}
-  {% endfor %}
-</nav>
+{# modules/article-page/views/show.html #}
+{% extends "layout.html" %}
+
+{% block main %}
+  <h1>{{ data.piece.title }}
+  <section>
+    {% area data.piece, 'main' %}
+{% endblock %}
 ```
 
-:::  tip Note:
-You won't see any choices for the filter unless you actually have products that have been assigned a color via the color field we just added. Similarly, if you add more than one filter, you will never see filter combinations that produce zero results.
+Instead of `data.page`, this template is using `data.piece` to access the piece data. In most other ways they work the same way as any other page template. There are some other special data available in show page templates:
```

Comment:
```suggestion
Instead of `data.page`, this template is using `data.piece` to access the piece data. There are some other special data available in show page templates:
```